src/clj/rems/api/user_settings.clj

```diff
@@ -0,0 +1,30 @@
+(ns rems.api.user-settings
+  (:require [compojure.api.sweet :refer :all]
+            [rems.api.schema :refer :all]
+            [rems.db.user-settings :as user-settings]
+            [rems.util :refer [getx-user-id]]
+            [ring.util.http-response :refer :all]
+            [schema.core :as s]))
+
+(s/defschema UpdateUserSettingsCommand
+  {(s/optional-key :language) s/Keyword})
+
+(s/defschema UserSettings
+  {(s/optional-key :language) s/Keyword})
```

Comment:
This can be made non-optional. Also the update command can be made so. Simplifies many things.

Reply:
I was probably thinking of some other settings that are potentially changed from somewhere else and kind of preparing for partial updating. But better to have it simple for now. Done.